common/plugins/eu.esdihumboldt.hale.common.align/help/classification/help.xhtml

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
+<title>Classification</title>
+</head>
+<body>
+<div id ="help">
+	<h4>Example mapping:</h4>
+	<p>
+		The classification function replaces values in the source data with new values. There are a variety of methods that can be used to classify source values. The Add source value button allows you to type values directly in the table. If the value entered is found in the source data, it will be classified to the value entered as target value. <br>
```

Comment:
Maybe "The classification function maps values from the source data to new values."?
I would at least avoid the "in the source data", as that sounds like the source data is modified.